android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/picker/ReactPickerManager.java

```diff
@@ -243,8 +241,8 @@ private View getView(int position, View convertView, ViewGroup parent, boolean i
 
       if (convertView == null) {
         int layoutResId = isDropdown
-              ? R.layout.simple_spinner_dropdown_item
-              : R.layout.simple_spinner_item;
+              ? android.R.layout.simple_spinner_dropdown_item
+              : android.R.layout.simple_spinner_item;
```

Comment:
hi. why was this change made with https://github.com/expo/expo/pull/13152/files#diff-80b4c7dfe6a55369097b7da73e07fa9ae93133e223eace1e78a35bfee00c490fR246-R247 reverted here a week later?
i have the problem, that the `numberOfLines` feature for multiline items are not working. with the help of @alesharik in https://github.com/react-native-picker/picker/pull/195#issuecomment-897118462 we tried to find out the reason. one can be the use of the android layouts here.

Reply:
I think this might have been a mistake - thanks for looking into this! I'll take a look today

Reply:
hi @ajsmth, any news about that? can i help somehow?

Reply:
hey there - sorry, I've been away for the past few days. we're going to try and get this into SDK 43. once I have a PR up I'll reference it here so you can see it

Reply:
@donni106 here is the PR: https://github.com/expo/expo/pull/14104